TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/PayloadAccessFaultingMap.java

```diff
@@ -7,8 +7,8 @@
 import java.util.Optional;
 import java.util.Set;
 
+import org.opensearch.migrations.IHttpMessage;
```

Comment:
Will probably be cleaner if you drop this here and adapt it later on, the same way that you'll likely adapt the reactor or java objects elsewhere.  Right now, this creates a hard dependency on the awsUtils library, which isn't/shouldn't be necessary for the rest of this package.

Reply:
Created an adapter